graylog2-web-interface/src/pages/LoginPage.jsx

```diff
@@ -67,7 +79,12 @@ const LoginPage = createReactClass({
       return (
         <div className="form-group">
           <Alert bsStyle="danger">
-            <a className="close" onClick={this.resetLastError}>×</a>{error}
+            <button className="close"
```

Comment:
eslint was complaining about this being an anchor without a `href`, so i swapped it to a button